omrs-database-polling-repository-event-mapper/build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+// Artifact names are taken from the directory by default, set in settings.gradle to override
+// The 'name' for the maven artifact, and description are set here
+ext.name = 'Abstract database polling event mapper connector'
+description = 'Abstract database polling event mapper connector'
+ext {
```

Comment:
the egeria pom is brought in as a list of managed dependencies, so you should find versions there. I think by default gradle will resolve to the highest of there or here.

Reply:
@planetf1  are you saying remove ```
jacksonVersion = '2.13.2'
    jupiterVersion = '5.8.2'
    springwebVersion = '5.3.19' 
```and not use those variables?

Reply:
II have removed these variables